README.rst

```diff
@@ -174,6 +174,18 @@ See `the contributing page <https://github.com/01org/cc-oci-runtime/blob/master/
 Building
 --------
 
+Acquiring
+~~~~~~~~~
+
+As ``cc-oci-runtime`` contains some golang code, the source tree needs to live
+in your ``$GOPATH`` in order to build. The easiest way to get the codebase into
+the correct place is with the command::
```

Comment:
Nope. That should be a new PR, for consistency in the doc. There are 14 or so other :: uses in this file, and no other code-blocks (and no other code-blocks in the whole repo that I can see).
So, I think we should merge as is with ::, and do another Issue/PR if we wish to move all the ::'s to code-blocks (and also decide if we do that for the other .rst files in the tree, which we probably should as there are only 2 others).